frontend/src/components/Modal.tsx

```diff
@@ -50,15 +51,17 @@ const Modal: React.FC<ModalProps> = ({
           {title}
         </DialogHeader>
         <Text className="mb-1 text-xs text-gray-700 dark:text-gray-300/60">
-          The issue summary and the recommended steps to address it have been generated by AI
+          The {entityType} summary and the recommended steps to address it have been generated by AI
```

Comment:
This is unique entirely for different types of modal (issue, event). Pass the full text instead of entity